src/core/src/lib.rs

```diff
@@ -145,25 +145,38 @@ impl<R: Resources> ShaderSet<R> {
     }
 }
 
-//TODO: use the appropriate units for max vertex count, etc
 /// Features that the device supports.
-#[allow(missing_docs)] // pretty self-explanatory fields!
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
 pub struct Capabilities {
-    pub max_vertex_count: usize,
-    pub max_index_count: usize,
+    /// Recommended maximum number of vertices per draw call.
+    /// If there is no recommendation, the value is `None`.
+    pub max_vertex_count: Option<VertexCount>,
```

Comment:
It seems to be only used in the gl backend and the refpages state that it's only a recommendation (above the limit the performance may decrease).
We could also switch to `VertexCount::MAX` instead of `Option` for the other backends.